### PR TITLE
Add new simulation type

### PR DESCRIPTION
**Pull request by ajlee21**
This PR address https://github.com/greenelab/ponyo/issues/37:

1. Adds a new simulation type to ponyo to allow users to select a template experiment that is not included in the compendium dataset that is used to train the VAE. 

Review `ponyo/simulate_expression_data.py` and `human_tests/Human_embed_shift_template_simulation.ipynb`

3. Modifies the github actions text notebooks to hopefully be faster to run.
Review `tests/` notebooks

### CODE REVIEW

**Comment by ajlee21** [comment]
Like, before, I ran and verified that the test notebooks ran successfully locally using pytest but they seem to be running for a while for now.

File: tests/train_vae_test.ipynb [hunk 1]
```diff
@@ -0,0 +1,283 @@
+{
```
Comment (taylorreiter) [inline tests/train_vae_test.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.    assert pd.read_csv(output_path, sep="\t")["val_loss"].values[-1] &lt; 2800, pd.read_csv(output_path, sep="\t")["val_loss"].values[-1]
</pre><p>just curious -- is this non-determinisitic? If its determinisitic, why not have == 2079.0769? Is that a rounded value or something?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/ponyo/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

**Review by taylorreiter — APPROVED** [review]
done reviewing notebooks, lgtm

File: tests/train_vae_test.ipynb (hunk 1 above)
Comment (ajlee21) [reply tests/train_vae_test.ipynb:1]: The VAE training is non-deterministic due to the randomness set in the background of python. Originally we checked for exact matching as you described but this didn't work.

File: human_tests/config_test_embed.tsv [hunk 2]
```diff
@@ -0,0 +1,29 @@
+base_dir	"../"
+local_dir	"../test_results/"
+raw_data_filename	"data/input/recount2_compendium_data_test.tsv"
+normalized_data_filename	"data/input/recount2_gene_normalized_data_test.tsv.xz"
```
[the remark this answers is not in the document]
Comment (ajlee21) [reply human_tests/config_test_embed.tsv:4]: Its a compression meant to preserve the quality of the original data and one that I saw python use which is why I selected it.

**Pull request merged**